mis_builder/models/mis_report.py

```diff
@@ -147,18 +147,17 @@ class MisReportKpi(models.Model):
 
     _order = "sequence, id"
 
-    def name_get(self):
-        res = []
+    @api.depends("description")
+    def _compute_display_name(self):
         for rec in self:
-            name = f"{rec.description} ({rec.name})"
-            res.append((rec.id, name))
-        return res
+            rec.display_name = f"{rec.description} ({rec.name})"
+            return rec
 
     @api.model
     def name_search(self, name="", args=None, operator="ilike", limit=100):
         domain = args or []
         domain += ["|", ("name", operator, name), ("description", operator, name)]
-        return self.search(domain, limit=limit).name_get()
+        return self.search(domain, limit=limit)._compute_display_name()
```

Comment:
In 17, the correct way to do this is to override `_name_search`.

Reply:
This override of _name_search is not correct. Actually, there is no need to override in this case. Defining `_rec_name_search = ['name', 'description']` in the model should be sufficient.